Heads-up: the sign-off on `8667686` failed after 45 min ([run 30659824103](https://github.com/spiceai/spiceai/actions/runs/30659824103)) on a one-line lint, and it will fail the same way on a re-dispatch — the offending line is still at HEAD.

```
error: item in documentation is missing backticks
    --> crates/runtime/tests/mysql/replication_e2e.rs:1169:31
1169 | /// Two columns stop short of MySQL's range because the Arrow type cannot hold
     |                               ^^^^^
     = note: `-D clippy::doc-markdown` implied by `-D clippy::pedantic`
error: could not compile `runtime` (test "integration") due to 1 previous error
make: *** [lint-rust] Error 101
```

Fix is `` `MySQL` ``. This is the `--tests` clippy pass that `make lint-rust` runs after the lib/bins pass, so a green `cargo clippy` on the lib alone won't catch it.

One detail that cost me time on a sibling PR earlier today: `gh run view <id> --log-failed` on a sign-off shows only the post-job cleanup, so the actual `error:` lines are visible only in the full `--log`.

<!-- fix-failing-prs: escalated signoff failing on clippy doc_markdown in tests pass -->

--- a/crates/runtime/tests/mysql/replication_e2e.rs
+++ b/crates/runtime/tests/mysql/replication_e2e.rs
@@ -1166,7 +1166,7 @@ const TYPES_CHECKS: &[(&str, i64)] = &[
 /// real server. Each value is asserted from the snapshot and again after a
 /// binlog INSERT, then an UPDATE covers the before/after row images.
 ///
-/// Two columns stop short of MySQL's range because the Arrow type cannot hold
+/// Two columns stop short of `MySQL`'s range because the Arrow type cannot hold
 /// it: `c_big_u` at `i64::MAX`, and `c_time` at the end of the day, since
 /// `Time64` is a time-of-day while `TIME` spans ±838 hours. Negative `TIME` is
 /// rejected with a structured error, covered by `negative_time_errors` in

